data/json/items/migration.json

```diff
@@ -1954,5 +1954,27 @@
     "id": "bag_paper_powder_small",
     "type": "MIGRATION",
     "replace": "bag_paper_powder"
+  },
+  {
+    "id": "steel_scissors",
+    "type": "MIGRATION",
+    "replace": "scissors"
+  },
+  {
+    "id": "teddy",
+    "type": "MIGRATION",
+    "replace": "toy_plush_teddy",
```

Comment:
This syntax doesn't work. If you want to replace an item with a specific variant, [there's syntax for it](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/JSON_INFO.md#obsoletion-and-migration).